src/stylesheets/themes/github-dark-orange/index.scss

```diff
@@ -9,16 +9,3 @@
 @import "./placeholder.scss";
 @import "./combobox.scss";
 @import "./code.scss";
-
```

Comment:
These codes should not be deleted. Because the current theme inherits the content of this theme. 

So to set the `background-color` of `body` in your theme, you only need to set `$bg-page` in the `variables.scss` to the value you want.